tests/gold_tests/h2/httpbin.test.py

```diff
@@ -83,7 +83,7 @@
 # Test Cases
 # ----
 
-# Test Case 0: Basic request and resposne
+# Test Case 0: Basic request and Response
```

Comment:
Looks like this is still "Response" with a capital R. Was that supposed to be lower-cased?

Thanks,
Brian

Reply:
You're absolutely right. Should be fixed in the latest version